src/pages/server/login.vue

```diff
@@ -36,10 +36,17 @@
         md="6"
         lg="5"
       >
-        <h1 v-if="!isEmpty(currentUser)" class="text-h4 mb-6 text-center">
-          {{ $t('login.loginAs', { name: currentUser.Name }) }}
-        </h1>
-        <h1 v-else class="text-h4 mb-6 text-center">{{ $t('login.login') }}</h1>
+        <v-col align="center" justify="center" dense class="mt-6 mb-3">
+          <h1 v-if="!isEmpty(currentUser)" class="text-h4 mb-3 text-center">
+            {{ $t('login.loginAs', { name: currentUser.Name }) }}
+          </h1>
+          <h1 v-else class="text-h4 mb-3 text-center">
+            {{ $t('login.login') }}
+          </h1>
+          <label class="server-used">{{
```

Comment:
We tend to prefer using Vuetify classes, as above. So you'd use the `text-center` class at least. For the opacity you can pick [here](https://vuetifyjs.com/en/styles/text-and-typography/#opacity) (I'd think `text--secondary` or `text--disabled` would suit you) and for the sizing something from [here](https://vuetifyjs.com/en/styles/text-and-typography/#typography) (so I'd guess something like `text-subtitle-1` for instance)